### PR TITLE
Update `golangci` file to re-enable the field alignment check

### PR DESCRIPTION
**Pull request by gab-arrobo**

### CODE REVIEW

**Comment by gab-arrobo** [comment]
@patriciareinoso, please your help reviewing this PR. Thanks!

**Review by patriciareinoso — APPROVED** [review]
LGTM

File: backend/auth/handlers_status_test.go [hunk 1]
```diff
@@ -19,10 +19,10 @@ func TestStatus(t *testing.T) {
 	AddAuthenticationService(router, mockJWTSecret)
 
 	testCases := []struct {
-		name         string
 		dbAdapter    dbadapter.DBInterface
-		expectedCode int
+		name         string
 		expectedBody string
+		expectedCode int
```
Comment (thakurajayL) [inline backend/auth/handlers_status_test.go:25]: What how this re-arrangement helps?

Comment (gab-arrobo) [reply backend/auth/handlers_status_test.go:25]: It helps to reduce/optimize memory footprint

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**